### PR TITLE
Created authorcard summary

### PR DESCRIPTION
**Pull request by pallavikadam-srijan**
created author card summary and made changes on authorcard  and authorcard column files as required

### CODE REVIEW

File: stories/Molecules/Text/Authors/Authorcolumn/authors.scss [hunk 1]
```diff
@@ -1,63 +1,45 @@
...
+      .author-wrapper__box {
+        width: 33%;
       }
-  
-      &.author-4-col {
-        width: calc(33% - 1rem);
+    }
+    &--4 {
+      .author-wrapper__box {
+        width: 25%;
```
Comment (surbhi90) [inline stories/Molecules/Text/Authors/Authorcolumn/authors.scss:21]: @pallavikadam-srijan It should have some space between the authors and only first and last (according to the grid that if column can have 3 or 4) should remove the spacing.

File: stories/Organism/Blocks/Authorcard/AuthorSummary.jsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+// return (
+//   <div className="author-card ">
+//     <Heading type="4" className="author-card__title" label={text} />
+//     <div className="author-card__summary left-right">
+//       <Author data={summarydata} width="1" image={image} />
+//       <P label={para} />
+//     </div>
+//   </div>
+// );
+// }
```
Comment (surbhi90) [inline stories/Organism/Blocks/Authorcard/AuthorSummary.jsx:21]: @pallavikadam-srijan Remove the commented code

File: stories/Organism/Blocks/Authorcard/AuthorSummary.jsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  viewport('.author-card');
+}, []);
+return (
+  <>
+  <Heading type="4" className="author-card__title" label={text} />
+  <div className="author-card left-right">
+    <div className="author-card__summary">
+      <Authorimg className="author-card__summary--image" image={image} variant= 'Large' />
+      <div className="author-card__summary--text"> 
+        <Heading type="2" label={Authorlabel} className="author-card__summary--text-name" />
```
Comment (surbhi90) [inline stories/Organism/Blocks/Authorcard/AuthorSummary.jsx:41]: @pallavikadam-srijan Remove the class and add the styles on heading tag

File: stories/Organism/Blocks/Authorcard/authorsummary.scss [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+//   //   }
+//   //   &__label {
+//   //     width: auto;
+//   //   }
+//   //   &__designation {
+//   //     font-size: 20px;
+//   //     line-height: 22px;
+//   //   }
+//   // }
+// }
```
Comment (surbhi90) [inline stories/Organism/Blocks/Authorcard/authorsummary.scss:32]: @pallavikadam-srijan Please remove commented code

File: stories/Organism/Blocks/Authorcard/authorsummary.scss [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+@import '../../../assets/scss/mixins';
+@import '../../../assets/scss/breakpoints';
+@import '../../../assets/scss/variables';
+
+.author-card {
+    border-top: 2px solid #000000;
```
Comment (surbhi90) [inline stories/Organism/Blocks/Authorcard/authorsummary.scss:39]: @pallavikadam-srijan Please use the variables

File: stories/Organism/Blocks/Authorcard/authorsummary.scss [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+        margin-top: 0;
+      }
+      &-name {
+        margin-bottom: 8px;
+      }
+    }
+  }
+  p {
+    @include devicebreak(medium) {
+      width: 448px;
```
Comment (surbhi90) [inline stories/Organism/Blocks/Authorcard/authorsummary.scss:64]: @pallavikadam-srijan why we are fixing height in px

**Review by surbhi90 — CHANGES_REQUESTED** [review]
Please check the comments

File: stories/Molecules/Text/Authors/Authorcolumn/authors.scss [hunk 7]
```diff
@@ -73,4 +109,69 @@
...
+        @include devicebreak(medium) {
+          .author-wrapper__box {
+    
+            &:nth-child(odd) {
+              margin-left: 1.875rem;
+              margin-right: unset;
+            }
+    
+            &:nth-child(even) {
+              margin-left: unset;
```
Comment (surbhi90) [inline stories/Molecules/Text/Authors/Authorcolumn/authors.scss:124]: @pallavikadam-srijan not sure if this property will work on all the browsers

File: stories/Molecules/Text/Authors/Authorcolumn/authors.scss [hunk 8]
```diff
@@ -1,63 +1,99 @@
...
+            margin-left: 1.875rem;
+          }
+        }
       }
     }
+
+    &--3 {
+      @include devicebreak(medium) {
+        .author-wrapper__box {
+          width: calc(32% - 1.875rem);
```
Comment (surbhi90) [inline stories/Molecules/Text/Authors/Authorcolumn/authors.scss:29]: @pallavikadam-srijan please use the grid system and don't write the same custom styles for this

**Review by surbhi90 — CHANGES_REQUESTED** [review]
please check the comment

File: stories/Molecules/Text/Authors/Authorcolumn/authors.scss (hunk 8 above)
Comment (pallavikadam-srijan) [reply stories/Molecules/Text/Authors/Authorcolumn/authors.scss]: done

**Review by surbhi90 — APPROVED** [review]

**Pull request merged**